rosplane/src/estimator_example.cpp

```diff
@@ -237,17 +242,19 @@ void estimator_example::estimate(const params_s &params, const input_s &input, o
 
     P_p_ = (A_d_*P_p_*A_d_.transpose() + Q_p_*pow(params.Ts/N_, 2));
   }
-
-    while(xhat_p_(3) > radians(180.0f)) xhat_p_(3) = xhat_p_(3) - radians(360.0f);
-    while(xhat_p_(3) < radians(-180.0f)) xhat_p_(3) = xhat_p_(3) + radians(360.0f);
+    
+    xhat_p_(3) = wrap(0.0, xhat_p_(3));
+    // while(xhat_p_(3) > radians(180.0f)) xhat_p_(3) = xhat_p_(3) - radians(360.0f);
```

Comment:
I would remove these commented out lines, since we won't be needing them anymore.

Reply:
Of course, I didn't realize they weren't deleted!